### ISSUE TITLE
media-query-list-comma-space-after conflict

### ISSUE THREAD

**Issue by andreyvolokitin** [shannonmoeller/stylelint-config-prettier#3]
[```media-query-list-comma-space-after``` rule](https://github.com/stylelint/stylelint-config-standard/blob/7b76d7d0060f2e13a331806a09c2096c7536b0a6/index.js#L78) from ```stylelint-config-standard``` has value of ```"always-single-line"``` which is in conflict with Prettier, so the value was changed to ```"always"``` in ```stylelint-config-prettier```. When Prettier breaks long media query lists to multiple lines there are no required spaces after comma (but just line breaks), which is then reported as error by Stylelint. Example CSS reformated by Prettier which is causing Stylelint error:
```css
@media print,
  (-webkit-min-device-pixel-ratio: 1.25),
  (min-resolution: 1.25dppx),
  (min-resolution: 120dpi) {
  /* Style adjustments for high resolution devices */
}
```

Seems like rules ```media-query-list-comma-newline-after``` and ```media-query-list-comma-space-after``` should somehow be aligned with Prettier (if it even possible), but I am still not sure how

**Comment by andreyvolokitin** [comment]
Temp fix (not sure about it): 
```json
"rules": {
    "media-query-list-comma-space-after": null
  }
```

**Comment by shannonmoeller** [comment]
If prettier is taking care of this case I think that disabling the rule is the right call. Thanks!

**Comment by shannonmoeller** [comment]
Fixed by 418af4e.
